misc/config_tools/configurator/packages/configurator/thirdLib/manager.py

```diff
@@ -110,7 +110,7 @@ def main():
     library_json = Path(__file__).parent / 'library.json'
 
     parser = argparse.ArgumentParser(
-        description='ACRN Configurator third part library manager.'
+        description='ACRN Configurator third party library manager.'
```

Comment:
ThirdLib/library.json has the same misspelling  (ACRN Third Part Library List)

Reply:
> ThirdLib/library.json has the same misspelling (ACRN Third Part Library List)

That's true for many files from this new configurator.